### DISCUSSION TITLE
Wait...

### DISCUSSION THREAD

**Discussion by ghost** [aflacc/ProjectFNF#40] [General]
When did Week 7 get here?

**Comment by actuallyasmartname** [comment]
dunno

**Comment by aflacc** [comment]
I forgot to say this on the github and only said it in the Discord:
> I am adding the week 7 assets purely to prepare for the update. None of it will be functional until the update is public. If you want to play the week the way its meant to be, do it in the Newgrounds. I will not support any mods that bring people away from the Newgrounds build. You will be able to download the week once the Newgrounds version is top result from googling Friday Night Funkin

**Reply by aflacc** [reply]
TL;DR is im just getting ready for the week 7 update

**Comment by ghost** [comment]
OOOOOHHHHH

**Reply by ghost** [reply]
IMMA COMPILE THIS CRAP AN D PLAY WEEK 7

**Reply by Heart-Artistic** [reply]
for fucks sake, the assets are just there TO PREPARE FOR THE UPDATE, compiling WONT add week 7

**Reply by UniversePM** [reply]
> IMMA COMPILE THIS CRAP AN D PLAY WEEK 7

Bro, can you read?

**Reply by ghost** [reply]
> > IMMA COMPILE THIS CRAP AN D PLAY WEEK 7
> 
> 
> 
> Bro, can you read?

yeah apparently not

**Reply by aflacc** [reply]
Feel free to use the assets. Pretty much everyone has them. I'm just not telling you anything about how to use them.

**Reply by ghost** [reply]
ah ok
